keyboards/quadrum/delta/keymaps/ansi/keymap.c

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2023 Sleepdealer
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_ansi(
+        KC_ESC,  KC_1,     KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,    KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  KC_DEL,   KC_MUTE,
+        KC_TAB,  KC_Q,     KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,  KC_PGUP,
+        KC_CAPS, KC_A,     KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,    KC_L,     KC_SCLN,  KC_QUOT,            KC_ENT,   KC_PGDN,
+        KC_LSFT, KC_NUBS,  KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,    KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,            KC_UP,
```

Comment:
`KC_LSFT, KC_NUBS,  KC_Z` would highlight that the ansi layout macro is wrong, as `KC_NUBS` would generally not feature on an ansi board.

Reply:
Why not just leave it in?
Can let the person decide if they want to use split LShift or not, even if it's usually not on an ANSI enter board.

Reply:
Because that is the general convention followed.

Reply:
Sure I guess
Changed